src/utils/selectors.ts

```diff
@@ -130,12 +131,16 @@ export const siblingVenuesSelector = (state: RootState) =>
   state.firestore.ordered.siblingVenues;
 export const venueEventsSelector = (state: RootState) =>
   state.firestore.ordered.venueEvents;
-export const subvenueEventsSelector = (state: RootState) =>
-  state.firestore.ordered.subvenueEvents;
 export const parentVenueEventsSelector = (state: RootState) =>
   state.firestore.ordered.parentVenueEvents;
-export const siblingVenueEventsSelector = (state: RootState) =>
-  state.firestore.ordered.siblingVenueEvents;
+export const makeSubvenueEventsSelector = (venueId?: string) => (
+  state: RootState
+): WithId<VenueEvent>[] | undefined =>
+  (state.firestore.ordered as any)[`subvenueEvents-${venueId}`];
+export const makeSiblingVenueEventsSelector = (venueId?: string) => (
+  state: RootState
+): WithId<VenueEvent>[] | undefined =>
+  (state.firestore.ordered as any)[`siblingVenueEvents-${venueId}`];
```

Comment:
Calling out use of `any` here, but not sure of a way to resolve it. So long as it doesn't 'escape' beyond this selector (which it shouldn't because of explicit return type) then should be ok I think.

Reply:
Yep- I suspect it won't since this is only for this one use-case; however, perhaps we can refactor later to pull it all into the same hook.